compiler/qsc_project/src/manifest.rs

```diff
@@ -0,0 +1,81 @@
+use crate::Error;
+use serde::Deserialize;
+use std::path::PathBuf;
+use std::{
+    env::current_dir,
+    fs::{self, DirEntry, FileType},
```

Comment:
Is this crate going to be what understands the qsharp.json and builds up the structure/context for general use by the compiler and language service? If so, it will need to run in wasm and the browser sandbox also. Be sure not to use anything from places like std::env and std::fs which will cause issues in that environment. It should use an interface with the host to do things like walk directories, read files, get the current dir/workspace, etc.